builder_requests.go

```diff
@@ -52,21 +52,21 @@ func (c *Client) BuilderFeatures() (*FeaturesResponse, error) {
 	return features, nil
 }
 
-// BuilderTemplate gets the function template for the provided Module and template name.
-func (c *Client) BuilderTemplate(module *Module, template string) (*EditorStateResponse, error) {
-	if module == nil {
-		return nil, errors.New("Module cannot be nil")
+// BuilderTemplate gets the plugin template for the provided Plugin and template name.
```

Comment:
```suggestion
// BuilderTemplate gets the plugin template for the provided plugin and template name.
```

Nit based on feedback in other repos.